src/tv2-common/helpers/graphics/pilot/create.ts

```diff
@@ -0,0 +1,33 @@
+import { IBlueprintActionManifest, IBlueprintAdLibPiece, IBlueprintPiece } from 'blueprints-integration'
+import { IsTargetingFull, IsTargetingOVL, PilotGraphicGenerator, PilotGraphicProps } from 'tv2-common'
+
+export function CreatePilotGraphic(
+	pieces: IBlueprintPiece[],
+	adlibPieces: IBlueprintAdLibPiece[],
+	actions: IBlueprintActionManifest[],
+	pilotGraphicProps: PilotGraphicProps
+) {
+	const { context, adlib, parsedCue } = pilotGraphicProps
+	if (
+		parsedCue.graphic.vcpid === undefined ||
+		parsedCue.graphic.vcpid === null ||
+		parsedCue.graphic.vcpid.toString() === '' ||
+		parsedCue.graphic.vcpid.toString().length === 0
+	) {
```

Comment:
What is achieved with this over `if (!parsedCue.graphic.vcpid)`?

Reply:
That's 3 years old code and according to the types, neither of these conditions makes sense anymore. I'll look into it while fixing one of my current gfx tasks.